spec/system/admin_company_value_spec.rb

```diff
@@ -0,0 +1,104 @@
+require 'rails_helper'
+require 'factory_bot_rails'
+
+RSpec.describe 'Modifying company values', type: :system do
+  let(:admin_user) { create(:admin_user) }
+  let!(:company_value) { create(:company_value) }
+
+  before do
+    driven_by(:rack_test)
+    sign_in admin_user
+  end
+
+  context 'when creating a new company value' do
+    it 'enables to create a new company value' do
+      expect(CompanyValue.count).to eq 1
+
+      go_to_form_new_company_value
+
+      fill_in 'Title', with: 'First Title'
+      click_button 'Update'
+
+      expect(CompanyValue.count).to eq 2
+      expect(page).to have_content 'Company Value was successfully created.'
+    end
+
+    it 'checks validation notices' do
+      expect(CompanyValue.count).to eq 1
+
+      go_to_form_new_company_value
+
+      fill_in 'Title', with: ''
+      click_button 'Update'
+
+      expect(CompanyValue.count).to eq 1
+      expect(page).to have_content "Title can't be blank"
+
+      fill_in 'Title', with: company_value.title
+      click_button 'Update'
+
+      expect(CompanyValue.count).to eq 1
+      expect(page).to have_content 'Title has already been taken'
+    end
+
+    def go_to_form_new_company_value
+      visit '/admin/pages/dashboard'
+      click_link 'Go to Company Values Admin Panel'
+      click_link 'New Company Value'
+    end
+  end
+
+  context 'when listing all company values' do
+    it 'enables to list all company values' do
+      company_value2 = create(:company_value, title: 'Second Company Value')
+
+      expect(CompanyValue.count).to eq 2
+
+      visit '/admin/company_values'
+
+      expect(page).to have_content company_value.title
+      expect(page).to have_content company_value2.title
+      expect(page).to have_content 'Edit'
+      expect(page).to have_content 'Destroy'
+    end
+
+    it 'unenabled to list without log in' do
```

Comment:
This spec is not really necessary - you should have other specs covering access already :)
```suggestion
    it 'does not allow to access page without loggin in as an admin' do
```